org.videolan.VLC.Plugin.fdkaac.json

```diff
@@ -0,0 +1,72 @@
+{
+  "id": "org.videolan.VLC.Plugin.fdkaac",
+  "runtime": "org.videolan.VLC",
+  "runtime-version": "stable",
```

Comment:
This is a comment for the VLC package that needs to be resolved first. `stable` is a useless version for extension points. Something like `1.6` is better since thats the runtime its based on and once VLC moves to `18.08` it is an ABI break and the version has to be bumped.

Reply:
you are sure it doesn't need to match the extended app version here ? I was trying with the default branch (master) and it didn't get injected into org.videolan.VLC context (which is versioned stable on flathub), with stable it is.

Reply:
Well yes. The version must match. So you add version to the VLC extension point and also add it here.

Reply:
Ah ok got it. Then I'll just use VLC version since the plugin depends on major VLC version being the same. Thx.

Reply:
good idea, done here and in the main repo.